hs-init.hs

```diff
@@ -70,7 +70,7 @@ defaultLicense :: Text
 defaultLicense = "MIT"
 
 defaultGHC :: Text
-defaultGHC = "8.2.2"
+defaultGHC = ""
```

Comment:
Maybe it's better to rename this variable as `defaultGHCs` since it represents multiple additional GHC versions to test with?

Reply:
I guess it's better just to remove it and put `""` right in the place where it's used as I'm sure we won't change his behavior. How about this?